integration-tests/js-compute/fixtures/app/src/btoa.js

```diff
@@ -737,7 +737,7 @@ routes.set('/btoa', () => {
     if (error) { return error; }
     error = assertThrows(() => atob("--"))
     if (error) { return error; }
-    error = assertThrows(() => atob("__"))
+    error = assertThrows(() => atob("--"))
```

Comment:
This is a duplicate of the above assertion I think

Reply:
This is actually a StarlingMonkey bug can you believe it....

Reply:
I've split this test out into it's own route to isolate this as a separate upstream issue.